samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-SequenceToSequence.ipynb

```diff
@@ -0,0 +1,852 @@
+{
```

Comment:
<p>On my local machine 1 epoch is taking approximately 3 hours. Is it possible to reduce the data size for faster processing? as it will be difficult to run the notebook for Try-it Live.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>